lib/rspec/support/object_formatter.rb

```diff
@@ -3,9 +3,23 @@ module Support
     # Provide additional output details beyond what `inspect` provides when
     # printing Time, DateTime, or BigDecimal
     module ObjectFormatter
+      class << self
+        attr_accessor :max_formatted_output_length
+      end
```

Comment:
Thank you for the recommendations, I agree it would be nicer to avoid adding state to the module.

* Should the max_formatted_ouput_length have a default value? If so, what should it be?
* Where should the shared instance be defined?
* I've used Forwardable for delegation but not to an instance - would it be possible to use Forwardable for that?